dom/src/main/scala-3/fs2/dom/Dom.scala

```diff
@@ -48,6 +52,190 @@ object Node {
+
+    def attributes(using F: Dom[F]): F[NamedNodeMap[F]] =
+      F.delay(node.attributes)
+
+    def nodeType: Int = node.nodeType
+
+    def previousSibling(using F: Dom[F]): F[Node[F]] =
+      F.delay(node.previousSibling)
+
+    def localName(using F: Dom[F]): F[String] = F.delay(node.localName)
+
+    def namespaceURI(using F: Dom[F]): F[Option[String]] = F.delay(Option(node.namespaceURI))
+
+    def textContent(using F: Dom[F]): Ref[F, String] =
+      new WrappedRef {
+        def unsafeGet() = node.textContent
+        def unsafeSet(s: String) = node.textContent = s
+      }
+
+    def parentNode(using F: Dom[F]): F[Node[F]] = F.delay(node.parentNode)
+
+    def nextSibling(using F: Dom[F]): F[Option[Node[F]]] = F.delay(Option(node.nextSibling))
+
+    def nodeValue(using F: Dom[F]): Ref[F, String] =
+      new WrappedRef {
+        def unsafeGet() = node.nodeValue
+        def unsafeSet(s: String) = node.nodeValue = s
+      }
+
+    def lastChild(using F: Dom[F]): F[Option[Node[F]]] = F.delay(Option(node.lastChild))
+
+    def childNodes(using F: Dom[F]): F[NodeList[F, Node[F]]] = F.delay(node.childNodes)
+
+    def nodeName: Option[String] = Option(node.nodeName)
+
+    def ownerDocument(using F: Dom[F]): F[Option[Document[F]]] = F.delay(Option(node.ownerDocument))
+
+    def lookupPrefix(namespaceURI: String)(using F: Dom[F]): F[Option[String]] =
+      F.delay(Option(node.lookupPrefix(namespaceURI)))
+
+    def isDefaultNamespace(namespaceURI: String)(using F: Dom[F]): F[Boolean] =
+      F.delay(node.isDefaultNamespace(namespaceURI))
+
+    def compareDocumentPosition(other: Node[F])(using F: Dom[F]): F[Int] =
+      F.delay(node.compareDocumentPosition(other))
+
+    def normalize(using F: Dom[F]): F[Unit] = F.delay(node.normalize())
+
+    def isSameNode(other: Node[F])(using F: Dom[F]): F[Boolean] = F.delay(node.isSameNode(other))
+
+    def contains(otherNode: Node[F])(using F: Dom[F]): F[Boolean] =
+      F.delay(node.contains(otherNode))
+
+    def hasAttributes(using F: Dom[F]): F[Boolean] = F.delay(node.hasAttributes())
+
+    def lookupNamespaceURI(prefix: String)(using F: Dom[F]): F[Option[String]] =
+      F.delay(Option(node.lookupNamespaceURI(prefix)))
+
+    def cloneNode(deep: Boolean)(using F: Dom[F]): F[Node[F]] = F.delay(node.cloneNode(deep))
+
+    def hasChildNodes(using F: Dom[F]): F[Boolean] = F.delay(node.hasChildNodes())
+
+    def insertBefore(newChild: Node[F], refChild: Node[F])(using F: Dom[F]): F[Node[F]] =
+      F.delay(node.insertBefore(newChild, refChild))
+
+    def baseURI(using F: Dom[F]): F[String] = F.delay(node.baseURI)
+
+    def isConnected(using F: Dom[F]): F[Boolean] = F.delay(node.isConnected)
+
+    def innerText(using F: Dom[F]): Ref[F, String] =
+      new WrappedRef {
+        def unsafeGet() = node.innerText
+        def unsafeSet(s: String) = node.innerText = s
+      }
+  }
+
+  val ENTITY_REFERENCE_NODE: Int = dom.Node.ENTITY_REFERENCE_NODE
```

Comment:
Not sure if we want to expose this here. One can just get it from dom. WDYT?

Reply:
I agree, probably better not to expose this. In scala-js-dom it is a native call, so storing it in a `val` here would be a performance/memory penalty.

https://github.com/scala-js/scala-js-dom/blob/51a807cc04ea134ece030d6d4005a8ce30c1de99/dom/src/main/scala/org/scalajs/dom/Node.scala#L177-L178